services/s3/pom.xml

```diff
@@ -42,6 +42,15 @@
                     </archive>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Should we add this in Kinesis service ?I see that in kinesis we have integration test related to a async clients .

Reply:
I'm not sure if it matters terribly much since all services need to support async clients. As the Javadoc mentions, it's admittedly somewhat arbitrary, as long as we have it somewhere. I'm fine with either location.

Reply:
> As the Javadoc mentions, it's admittedly somewhat arbitrary, as long as we have it somewhere
Looks fine, then I think no need to move to kinesis.